cmd/fyne/commands/bundle.go

```diff
@@ -1,7 +1,7 @@
-// Package commands provides functionality for managing fyne packages and the build process
```

Comment:
Should this have been removed?

Reply:
No, sorry. Got wiped out when I moved around a few files.